### ISSUE TITLE
problem: content boxes have different sizes

### ISSUE THREAD

**Issue by gazhayes** [EmurgoHK/cardanoupdate#221]
problem: content boxes have different sizes

possible solution:

Use a wide box instead of a tall box.

Not to show all words but limited the words(eg:50 words) shown in every boxes look better and let user digest all notices in one glance, user can further click in it(put “ show more…” at the end of content) if they want to deeper understand. 

<img width="688" alt="screenshot 2018-10-23 at 7 00 48 pm" src="https://user-images.githubusercontent.com/8143945/47356209-f88b2400-d6f5-11e8-8433-39596fcf736d.png">

**Comment by omar20261** [comment]
@emurgobot claim

**Comment by omar20261** [comment]
@emurgobot done

**Comment by gazhayes** [comment]
Yeah, I was reffering to all content (site wide), especially the home route, not just the projects page.
